### DISCUSSION TITLE
MONAILabel Active Learning is not creating new iterations in Endoscopy app

### DISCUSSION THREAD

**Discussion by bilalcodehub** [Project-MONAI/MONAILabel#1295] [General]
Hi,

I completed a task in `CVAT` generated by `MONAILabel` server. There are 25 images in the dataset folder. The job only picks one image based on the start_server settings. I clicked `Save` in `CVAT` UI as well as finished the job. The name of the job is prefixed with `DONE_` which means it is marked complete as shown below:
<img width="1030" alt="image" src="https://user-images.githubusercontent.com/7869659/218251298-b7e7e11b-46bc-4a9b-a4a5-45dec9a04dbc.png">

Now, I am expecting a second iteration but the MONAILabel log shows the following message:
```
[2023-02-11 09:19:21,024] [2012] [MainThread] [INFO] (lib.scoring.cvat:76) - Latest Active Learning Task in CVAT is under progress/not-consumed. Skip to create new!
```
This means that MONAILabel is not informed of the completed task. What I will be doing wrong? I tried checking CVAT UI options but didn't find anything useful. Do I have to upload annotations to let the label server know about this?

**Comment by bilalcodehub** [comment]
I see that `MONAILabel` is creating a mask for the completed task inside `datasets` folder under `labels/final` path. If I download masks from CVAT it properly labelled as done by `tooltracking` model, but the one added to the datasets folder by `MONAILabel` is blank mask containing black pixels only as shown below:
<img width="1472" alt="image" src="https://user-images.githubusercontent.com/7869659/218252511-f304f4ea-651d-4135-bc8c-48ed30c7785d.png">

Mask downloaded from 'CVAT' using Export dataset option (though images are different):
<img width="1468" alt="image" src="https://user-images.githubusercontent.com/7869659/218253628-c47ab70b-4fee-42e0-b650-5228f5fbde3c.png">

**Comment by SachidanandAlle** [comment]
these are PNG images which label values 1,2,3 kind.. so you will not see them in normal viewers.. you can try to load them in 3D slicer or MITK workbench or ITKSNap kind.. the value of pixels (labels) are between 1-255
